include/vapor/DCBOV.h

```diff
@@ -0,0 +1,175 @@
+#pragma once
+
+#include <vector>
+#include <algorithm>
+#include <map>
+#include <iostream>
+#include <vapor/MyBase.h>
+#include <vapor/NetCDFCFCollection.h>
+#include <vapor/Proj4API.h>
+#include <vapor/UDUnitsClass.h>
+#include <vapor/utils.h>
+#include <vapor/DC.h>
+#include <vapor/BOVCollection.h>
+
+namespace VAPoR {
+
+class BOVCollection;
+
+//!
+//! \class DCBOV
+//! \ingroup Public_VDCBOV
+//!
+//! \brief Class for reading a "Brick of Values"
```

Comment:
Also identify which elements are not supported by this class.

Reply:
Fixed.